defaults/main.yml

```diff
@@ -3,7 +3,7 @@ py_version: 2.7.9 # same as AWS
 
 # executable names
 executable_easy_install: easy_install-2.7
-executable_pip: pip2.7
+executable_pip: pip-2.7
```

Comment:
? I think this will break the vagrant provisioning. It used to `pip-2.7` on the original playbooks, but I unintentionally broke behavior by "fixing" it to what it is now. @tjoelsson fixed the playbooks by providing overrides on the pertinent inventory/hosts files.